docs/tutorials.rst

```diff
@@ -0,0 +1,24 @@
+Tutorials
```

Comment:
What would be the requirements for a topic to move from shared knowledge in the forum to tutorial? 

It could be valuable to define that here. That way it would make it easier for people to propose PRs for the tutorial section.

Reply:
For me the big difference is that tutorials should be sufficiently stabilized to be added to the documentation. Maybe the right workflow could be:

1. Write a tutorial in the forums (in a new subcategory)
2. Get feedback from the community and adjust the tutorial
3. Once the tutorial has been battle-tested by a few users, add it to the docs.

Reply:
Sounds good to me